playbooks/common/openshift-glusterfs/config.yml

```diff
@@ -1,6 +1,6 @@
 ---
-- name: Open firewall ports for GlusterFS
-  hosts: oo_glusterfs_to_config
+- name: Open firewall ports for GlusterFS nodes
+  hosts: glusterfs
```

Comment:
My concern is that we're using byo groups here and we should be using oo_* groups exclusively in playbooks/common but it looks like we'd need to do a bit of refactoring to get us there based on the group mapping logic that sets gluster_registry = glusterfs hosts if gluster_registry group is empty. I'll open an issue for us to track this but we'll take it as is for now.